src/style.css

```diff
@@ -0,0 +1,140 @@
+/* rerion reset-css */
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
font should be moved to html selector tag. And `p` and `body` should be combined

Reply:
fix it

Reply:
Move font on separate html selector and add body here